src/pipecat/services/google/stt.py

```diff
@@ -747,6 +747,11 @@ async def _stream_audio(self):
         try:
             while True:
                 try:
+                    if self._request_queue.empty():
+                        # wait for 10ms in case we don't have audio
+                        await asyncio.sleep(0.01)
+                        continue
+
                     # Start bi-directional streaming
                     streaming_recognize = await self._client.streaming_recognize(
                         requests=self._request_generator()
```

Comment:
Without the above, we keep receiving this error until the user finallly connects to the call:
```
2025-06-12 12:03:17.872 | ERROR    | pipecat.services.google.stt:_process_responses:852 - Error processing Google STT responses: 409 Stream timed out after receiving no more client requests.
2025-06-12 12:03:17.873 | WARNING  | pipecat.services.google.stt:_stream_audio:774 - GoogleSTTService#0 Reconnecting: 409 Stream timed out after receiving no more client requests.
```